crew

```diff
@@ -84,80 +49,6 @@ end
 @opt_verbose = args["--verbose"]
 @opt_src = args["--build-from-source"]
 
-# colorization
-class String
-  def colorize(color_code, shade)
-    "\e[#{shade};#{color_code}m#{self}\e[0m"
-  end
-
-  def black
-    colorize(30, 0)
-  end
-
-  def red
-    colorize(31, 0)
-  end
-
-  def green
-    colorize(32, 0)
-  end
-
-  def orange
-    colorize(33, 0)
-  end
-
-  def blue
-    colorize(34, 0)
-  end
-
-  def purple
-    colorize(35, 0)
-  end
-
-  def cyan
-    colorize(36, 0)
-  end
-
-  def lightgray
-    colorize(37, 0)
-  end
-
-  def gray
-    colorize(30, 1)
-  end
-
-  def lightred
-    colorize(31, 1)
-  end
-
-  def lightgreen
-    colorize(32, 1)
-  end
-
-  def yellow
-    colorize(33, 1)
-  end
-
-  def lightblue
-    colorize(34, 1)
-  end
-
-  def lightpurple
-    colorize(35, 1)
-  end
-
-  def lightcyan
-    colorize(36, 1)
-  end
-
-  def white
-    colorize(37, 1)
-  end
-end
-
-#disallow sudo
```

Comment:
You removed sudo check also.  Please revert this part.  Thanks.

Reply:
Never mind, you moved sudo check to different place.  I see.

Reply:
I moved it before the docopt check. Sorry I didn't say anything. Before, if you wrote a wrong command as sudo it would say the command was wrong instead of complaining you're sudo.

If you prefer the old way I can revert it.